2013/P_U2/main.cpp

```diff
@@ -0,0 +1,81 @@
+#include <iostream>
+#include <fstream>
+#include <vector>
+using namespace std;
+
+struct Savivaldybe{
+    string pavadinimas;
+    int mokykluSk = 0;
+};
+
+struct UnikaliSavivaldybe{
```

Comment:
What is the difference between those two? Maybe it is better to rename one of them?